CHECKLIST

```diff
@@ -24,6 +24,8 @@ The topics can be an empty list, and the difficulty level is from 1 to 10, and c
 ]
 ```
 
+The order of the exercises is decided by the order in this "exercises" array. Try to make them roughly increasing in difficulty, but don't worry too much—we can always adjust the order later, as we get feedback from people doing the exercises.
```

Comment:
I like the wordsmithing. I'll take it.

We don't have a dedicated feedback mechanism, but people do give us feedback.

I don't know if we'll sort things programmatically. I've been working with the UX folks on some of these questions. When I've got my head wrapped around it, I'll talk about it more.